SpiffWorkflow/version.py

```diff
@@ -1,4 +1,4 @@
 """
 Warning: This file is automatically generated.
 """
-__version__ = 'v0.3.0.j1.15.0a3'
+__version__ = 'v0.3.0-j5.28.0a2'

```

Comment:
Hmm, this was a full stop but now it's a hyphen. I wonder if that will be a problem.

Reply:
We're using the new format in `master` and it seems to work fine

Reply:
I changed it to use a . already just in case.